licenses/github.com/BurntSushi/xgb/xgbgen/COPYING

```diff
@@ -0,0 +1,13 @@
+            DO WHAT THE FUCK YOU WANT TO PUBLIC LICENSE
```

Comment:
The attorney's love this license.. ;-)

Reply:
I am a bit confused by this since I don't think we should have this, but its explicitly whitelisting `github.com/BurntSushi/xgb` in our config. I am not sure why...

Its not strictly related to this PR - we have the same in istio/istio already and pretty much every other repo. So I am not sure why its whitelisted. K8s depends on it as well.

I vaguely recall that our license check is more aggressive than kubernetes, and they don't "count" code that is not actually part of the final binary (eg test dependencies), whereas our linter is pulling everything from go modules

Reply:
I recall having a discussion on this one earlier with Martin. Have to see if it was in a PR or more than likely lost in Slack.

Reply:
IMO this license may be fine for most big firms. I was just making a joke, but I guess I shouldn't do that in a PR.